pkg/apis/pytorch/v1/defaults.go

```diff
@@ -57,10 +57,32 @@ func setDefaultPort(spec *v1.PodSpec) {
 	}
 }
 
+func setElasticPolicy(job *PyTorchJob) {
+	if job.Spec.ElasticPolicy != nil {
+		if job.Spec.ElasticPolicy.MaxReplicas != nil &&
+			job.Spec.ElasticPolicy.MinReplicas != nil {
+			return
+		} else if job.Spec.ElasticPolicy.MaxReplicas != nil {
+			// Set MinRepliacs to elasticPolicy.MaxReplicas.
+			job.Spec.ElasticPolicy.MinReplicas = job.Spec.ElasticPolicy.MaxReplicas
+		} else if job.Spec.ElasticPolicy.MinReplicas != nil {
+			job.Spec.ElasticPolicy.MaxReplicas = job.Spec.ElasticPolicy.MinReplicas
```

Comment:
The default behavior here is to avoid potential damage to the cluster. Personally, I prefer not to enable elastic if the users do not set the Min and Max.

I think we can document it to keep users know it. WDYT